packages/react-native-reanimated/android/src/main/cpp/reanimated/CMakeLists.txt

```diff
@@ -19,6 +19,7 @@ target_include_directories(
     "${ANDROID_CPP_DIR}"
     "${REACT_NATIVE_DIR}/ReactCommon"
     "${REACT_NATIVE_DIR}/ReactAndroid/src/main/jni/react/turbomodule"
+    "${REACT_NATIVE_DIR}/ReactCommon/react/nativemodule/core/ReactCommon"
```

Comment:
Why do we need it now with those changes?

Reply:
Fixed in [22793d2](https://github.com/software-mansion/react-native-reanimated/pull/6539/commits/22793d24d0b5455ef915942911a82043dc419925)